com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/archive/test/ZipPublisherTest.java

```diff
@@ -0,0 +1,119 @@
+package com.dynamo.bob.archive.test;
+
+import com.dynamo.bob.archive.ArchiveEntry;
+import com.dynamo.bob.archive.publisher.PublisherSettings;
+import com.dynamo.bob.archive.publisher.ZipPublisher;
+import com.dynamo.liveupdate.proto.Manifest;
+import org.junit.Test;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ForkJoinPool;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class ZipPublisherTest {
+// This isn't a test really, this is setup for development to be able to iterate faster on the archiving
+// see https://github.com/defold/defold/issues/10012
+//    @Test
```

Comment:
I’m keeping it for the future in case we need to improve it even more.  
It’s much easier to have it as a test and use it from IntelliJ IDEA (no need to rebuild Bob all the time).